src/scripts/region-redirects.js

```diff
@@ -0,0 +1,159 @@
+import Cookies from 'js-cookie';
+import config from '../../regions.config';
+// import datadogLogs from './components/dd-browser-logs-rum';
+
+const allowedRegions = ['us', 'eu'];
```

Comment:
can we pull this in from the config file as well? This way, when new regions are added, we don't have to change JS and can update everything from config